VERSION.cmake

```diff
@@ -1,6 +1,6 @@
 set (DEFAULT_LIBDNF_MAJOR_VERSION 0)
 set (DEFAULT_LIBDNF_MINOR_VERSION 54)
-set (DEFAULT_LIBDNF_MICRO_VERSION 0)
+set (DEFAULT_LIBDNF_MICRO_VERSION 1)
```

Comment:
Sorry for not mentioning this the first time. Bump the MINOR version instead of MICRO when adding interface?
I've also mentioned this before, so if you for some reason prefer to bump MICRO in these cases, we can discuss it :slightly_smiling_face:

Reply:
I thought we agreed on bumping micro in general and then minor when doing a release (if there was a feature since the last release) instead of bumping it for every feature. But we can discuss it.

Reply:
Okay.. I think I either forgot or am misremembering... If that's so, it's fine.